packages/mobile/src/screens/profile-screen/ProfileHeader/ExpandedSection.tsx

```diff
@@ -66,7 +72,9 @@ export const ExpandedSection = () => {
       <SocialsAndSites />
       <View style={{ flexDirection: 'row', marginVertical: spacing(2) }}>
         <ProfileTierTile />
-        {isOwner ? null : <ProfileMutualsButton />}
+        {isOwner || current_user_followee_follow_count === 0 ? null : (
```

Comment:
nice

Reply:
yeah! checked with julian on it too. 2 birds